installing/installing_vsphere/preparing-to-install-on-vsphere.adoc

```diff
@@ -0,0 +1,76 @@
+[id="preparing-to-install-on-vsphere"]
+= Preparing to install on vSphere
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-vsphere
+
+toc::[]
+
+
+[id="preparing-to-install-on-vsphere-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+* You provisioned xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] for your cluster. To deploy a private image registry, your storage must provide
```

Comment:
This prerequisite is relevant I think only to some of the installation methods for vSphere. Or at least it is only included in some of the installation assemblies.

Reply:
Removed this from the page, thank you!